resources/views/messages/reminder.blade.php

```diff
@@ -0,0 +1 @@
+<?php echo $content->body ?>
```

Comment:
Can't use blade? Or is this just placeholder for now?

If not, then probs don't need the `.blade` in the file name...